tests/helixprep.proj

```diff
@@ -0,0 +1,152 @@
+<Project ToolsVersion="12.0" DefaultTargets="ArchiveAll" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(ToolsDir)\net45\Microsoft.DotNet.Build.Tasks.dll"/>
+  
+  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
+
+  <PropertyGroup>
+    <DiscoveryDirectory>$(TestWorkingDir)</DiscoveryDirectory>
+    <CoreRootDir Condition="'$(RuntimeID)' == '' ">$(CORE_ROOT)</CoreRootDir>
+    <CoreRootDir Condition="'$(RuntimeID)' != '' ">$(CORE_OVERLAY)</CoreRootDir>
+    <CoreRootName Condition="'$(RuntimeID)' == '' ">Core_Root_Windows_NT-$(__BuildArch)</CoreRootName>
+    <CoreRootName Condition="'$(RuntimeID)' != '' ">Core_Root_$(RuntimeID)</CoreRootName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TestCmds Include="$(DiscoveryDirectory)\**\*.cmd" ></TestCmds>
+    <XunitDlls Include="$(DiscoveryDirectory)\*\*.XUnitWrapper.dll" ></XunitDlls>
+  </ItemGroup>
+
+  <!-- Build the platform-specific executable to run an individual xunit wrapper -->
+
+  <Target Name="GenerateWrapperExecutables"
```

Comment:
Two comments:  
1) This looks like you're generating wrapper scripts, not executables
2) Do you build once for both Linux + WIndows, or have some other reason to generate both bash and cmd scripts?